jx_sqlite/query_table.py

```diff
@@ -278,7 +278,7 @@ def query(self, query):
 
             output = Data(
                 meta={"format": "table"},
-                header=column_names,
+                header=[split_field(h)[0] if "\\" in h else h for h in column_names],
```

Comment:
Your solution does not sit right with me; checking for a backslash is standing in for some other piece of logic.  I will check these two tests and get back to you with a better answer.